package.json

```diff
@@ -4,11 +4,11 @@
   "public": false,
   "private": true,
   "scripts": {
-    "start": "gulp dev:extension",
-    "mascara": "gulp dev:mascara & node ./mascara/example/server",
-    "dist": "gulp dist",
+    "start": "npm run build:ts && gulp dev:extension",
```

Comment:
This was hastily done so TS files are compiled before main NPM tasks, but I was only testing with `start` and `test`. Ideally we'd revisit and greatly simplify our toolchain, or at the very least, compile TypeScript using a gulp task.